src/journal/models.py

```diff
@@ -586,6 +600,18 @@ def path_parts(self):
         return self.FILES_PATH, self.issue.pk
 
 
+class IssueEditor(models.Model):
+    account = models.ForeignKey('core.Account')
+    issue = models.ForeignKey(Issue)
+    role = models.CharField(max_length=255, default='Guest Editor')
```

Comment:
This can be confused easily with account roles, but I can't think of a better name

Reply:
Me either.